app/views/Dashboard/Overview/MapView/index.tsx

```diff
@@ -366,10 +352,13 @@ function MapView(props: MapViewProps) {
                 id: +indicatorValue.countryId,
                 value: indicatorValue.indicatorValue ?? 0,
                 iso: indicatorValue.iso3,
+                format: indicatorValue.format,
             }),
         )
             .filter((item) => item.value > 0);
-        return countryIndicator ?? [];
+        const sortedData = [...(countryIndicator ?? [])];
+        sortedData.sort((a, b) => compareNumber(a.value, b.value, -1));
+        return sortedData;
```

Comment:
No need to make a copy as `map` will already create a copy of the list.
We can easily do list.map().sort()

Reply:
Ok